sql/src/main/java/io/crate/operation/reference/sys/cluster/ClusterSettingsExpression.java

```diff
@@ -141,7 +142,7 @@ public ClusterSettingsExpression(Settings settings, ClusterService clusterServic
 
         //ApplySettings applySettings = new ApplySettings(settings, values);
         //nodeSettingsService.addListener(applySettings);
-        applyInitialSettingsAndRegisterUpdateConsumer(CrateSettings.SETTINGS, settings);
+        applyInitialSettingsAndRegisterUpdateConsumer(CrateSettings.CRATE_SETTINGS, settings);
```

Comment:
This means `select settings from sys.cluster` Won't show updates on dynamic ES settings?